src/CTA.WebForms/FileConverters/CodeFileConverter.cs

```diff
@@ -49,10 +49,15 @@ public class CodeFileConverter : FileConverter
                 LogHelper.LogError(e, $"{Rules.Config.Constants.WebFormsErrorTag}Exception occurred when trying to retrieve semantic model for the file {FullPath}. " +
                                       "Semantic Model will default to null.");
             }
-
+            
             if (_fileModel != null)
             {
-                _classConverters = classConverterFactory.BuildMany(RelativePath, _fileModel);
+                _classConverters = _classConverterFactory.BuildMany(RelativePath, _fileModel);
+            }
+            else
+            {
+                _classConverters = new List<ClassConverter>();
+                LogHelper.LogWarning($"Semantic model was not found so class conversion cannot take place. Returning zero class converters for file: {FullPath}");
```

Comment:
why this log level is warning? if there are no class convertor in the file that we support, do we log it as info?

Reply:
I logged it as a warning because in this case, _classConverters being empty is a side effect of an error condition (missing semantic model). 

It's possible that 0 class converters would have been returned anyway (in which case nothing would be logged), but the warning signifies that particular case might be worth looking into.

Reply:
makes sense. Approved :)